examples/hl_lhc_collision/reference/main.mask

```diff
@@ -3,7 +3,8 @@
 !*************************!
 
 ! Check documentation at http://lhcmaskdoc.web.cern.ch/mainmask/
-system, "ln -fns /afs/cern.ch/eng/tracking-tools tracking_tools";
+!system, "ln -fns /afs/cern.ch/eng/tracking-tools tracking_tools";
+system, "ln -fns /afs/cern.ch/work/f/fvanderv/projects/MaskDevelopment tracking_tools";
```

Comment:
@freddieknets 
Why are you putting your folder there
```suggestion
system, "ln -fns /afs/cern.ch/work/f/fvanderv/projects/MaskDevelopment tracking_tools";
```

Reply:
Oversight from my side, sorry! Fixed it.